.github/workflows/pa11y.yml

```diff
@@ -0,0 +1,28 @@
+name: pa11y tests
+
+on: [pull_request]
```

Comment:
as this check runs against the deployed version it doesn't make a lot of sense to check on each pull request. instead could you set this up to run every night using cron?

Reply:
Instead of running against the live url, I've changed it to run on the version in the pull request.